test/widgets/app_test.dart

```diff
@@ -51,4 +54,56 @@ void main() {
       ]);
     });
   });
+
+  group('choose account page', () {
```

Comment:
nit:
```suggestion
  group('ChooseAccountPage', () {
```

Using the identifier that's also found in the code under test helps make it easy to find the test from the code tested and vice versa.